README.md

```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
 
 | Field Name              | Type                | Description |
 | ----------------------- | ------------------- | ----------- |
-| classification:classed  | `Classed`           | **REQUIRED**. Classes in the dataset |
+| classification:class_list  | `[string]`           | **REQUIRED**. Summary of class names in the dataset |
```

Comment:
I assume this is holding the same values as the name in classification:classes? Why do we need a summary then?

Reply:
We were discussing that in #4, probably don't need it but I'm waiting for clarification from @pjhartzell

Reply:
@drwelby I'm good with removing `classification:class_list`